app/NEUFriends/models.py

```diff
@@ -0,0 +1,37 @@
+import uuid
+from django import forms
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    username = models.CharField(max_length=30)
+    password = models.CharField(max_length=50)
+    email = models.EmailField(max_length=70,blank=True,unique=True)
+    bio = models.CharField(max_length=100)
+    friends = []
+
+    def __str__(self):
+        return self.username
+
+EVENT_CATEGORIES = (
+    ('Sports','Sports'),
+    ('Concert', 'Concert'),
+    ('Movie','Movie'),
+    ('Conference','Conference'),
+    ('Festival','Festival'),
+    ('Community','Community'),
+)
+class Event(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=30)
+    category = models.CharField(max_length=10, choices=EVENT_CATEGORIES)
```

Comment:
There are a couple of options for this:
1. Mongo-specific route:[ We can use `djongo`'s `ListField`](https://www.djongomapper.com/using-django-with-other-fields/#listfield) (I think it may have already been deprecated in favor of JSONField, so you should try that first if you go this route). The downside of this is that both `ListField` and `JSONField` aren't natively supported by the admin console - you either have to write a custom view (which I don't think would be that hard), or just go without it (which also wouldn't be that bad, since we probably won't be using the admin console that much). 
2. Strings: Like we discussed on call, this could also just be a string separated by some value, i.e. `category1|category2|...`. We don't really care about performance, and users can't input their own categories so there's no security risks either. In the admin console, we'd just have to be sure to always follow this format.

Let me know what you think